web-app-sql-docdb-search/README.md

```diff
@@ -0,0 +1,9 @@
+# Provision a web app with a SQL Database, DocumentDB and Azure Search
+
+

```

Comment:
Since this template shows off platform capabilities, I'd suggest renaming this to 201-web-app-sql-docdb-search